test/jdk/java/util/regex/RegExTest.java

```diff
@@ -62,6 +61,8 @@
 import java.util.stream.IntStream;
 import java.util.stream.Stream;
 
+import static java.lang.Character.*;
```

Comment:
These static imports make the uses of the methods harder to recognize as from the Character class.